rome-modules/src/test/java/com/rometools/modules/mediarss/MediaModuleTest.java

```diff
@@ -91,6 +89,30 @@ public void testParse() throws Exception {
         }
     }
 
+
+    public void testParseFileSizeWithUnits() throws Exception {
+        final SyndFeed feed = getSyndFeed("org/rometools/feed/module/mediarss/filesize-with-unit.xml");
```

Comment:
Did exactly this in the last commit. This test became an really ugly variant of me wanting to copy how it's done elsewhere in the project with multiple asserts and wanting to test some edge cases due to the inherent implicit complexity of regexes :p